docs/source/rule-testing.rst

```diff
@@ -76,6 +76,84 @@ For example, to replace a time based field with ``last_hour``:
   }
 
 
+Validating Log Schemas
+~~~~~~~~~~~~~~~~~~~~~~
+
+In some cases, there may be logs coming into StreamAlert that are of a known type, but you do not have a current need to apply rule
+logic to them. However, it is still advantageous to add schemas for these logs and *test* to ensure your new schemas are valid.
+
+This is possible through the addition of log schema(s) to ``conf/logs.json`` and creation of test record(s) in ``tests/integration/rules/``
```

Comment:
`This is possible by first adding the new schema(s) to ``conf/logs.json`` along with creation of test record(s)...`